docs/releases/maintenance-policy.md

```diff
@@ -1,30 +1,31 @@
 # Maintenance policy
 
-Our goal is to ensure that the latest InvenioRDM release is supported with bug and security fixes for minimum one year after the release date, and possibly longer.
-We aim at having a new release every year. We strive our best to ensure that upgrades between versions are fairly straight-forward to ensure users follow our latest releases.
+Our goal is to ensure that the latest InvenioRDM release is supported with bug and security fixes for a minimum of one year after its release date, and 6 months after the next latest release. This promotes stability and gives adopters a dependable transition period. We aim at having a new release every year. We strive our best to ensure that upgrades between versions are straight-forward to ensure users follow our latest releases.
```

Comment:
For me, the important part is to provide a concrete time-frame (and then stick to it). Whether it's 6 months or 3 months or something else can be confirmed here. I am putting 6 months because it's what the previous language about LTS was saying in some places. (Personally 3 months would be fine, as long as there **is** an overlap support period when a new version is coming out).

A couple of implications of the above policy:
- if next release is more than a year after the previous one, then the overall support period of the previous one will have been more than 1.5 yrs
- if next release is less than a year after the previous one, then the overall support period of the previous one will have been less than 1.5yr but still at least 1 year.